CONTRIBUTING.md

```diff
@@ -21,12 +27,17 @@ new to this, you may also want to read the [Tao of the IETF](https://www.ietf.or
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+# Engaging with the QUIC community
+
+The QUIC Working Group scope of work is described in our charter and it extends beyond the development of the documents held in this repository. Anyone is welcome to contribute to the QUIC community; you don't have to join the Working Group, because there is no "membership" -- anyone who participates in the work **is** a part of the QUIC Working Group.
```

Comment:
```suggestion
The scope of work in the QUIC Working Group is described in our charter and extends beyond the development of the documents in this repository. Anyone is welcome to contribute to the QUIC Working Group; you don't have to join the Working Group, because there is no "membership" -- anyone who participates in the work **is** a part of the QUIC Working Group.
```